### PR TITLE
Stabilize attributes on generic parameters

### PR DESCRIPTION
**Pull request by petrochenkov**
Closes https://github.com/rust-lang/rust/issues/48848

### CODE REVIEW

**Comment by nikomatsakis** [comment]
Blocked on FCP in #48848

**Comment by pietroalbini** [comment]
The FCP in #48848 ended. Can you review this @nikomatsakis?

Also @petrochenkov, there are some merge conflicts you should fix.

**Review by nikomatsakis — APPROVED** [review]

**Comment by nikomatsakis** [comment]
r=me post rebase

File: src/libsyntax/feature_gate.rs [hunk 1]
```diff
@@ -566,6 +563,8 @@ declare_features! (
     (accepted, match_default_bindings, "1.26.0", Some(42640), None),
     // allow `'_` placeholder lifetimes
     (accepted, underscore_lifetimes, "1.26.0", Some(44524), None),
+    // Allows attributes on lifetime/type formal parameters in generics (RFC 1327)
+    (accepted, generic_param_attrs, "1.26.0", Some(48848), None),
```
Comment (kennytm) [inline src/libsyntax/feature_gate.rs]: Oops. This should be 1.27 now.

**Pull request merged**